### PR TITLE
added effect in OUR SERVICES section

### PR DESCRIPTION
**Pull request by yatigautam**
## Description

added a tilting hover effect in OUR SERVICES section.

## Reviewers

@tech-bash

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -126,28 +126,49 @@ export default function Home() {
...
               <div>
-                <Image loading="lazy" src={assign} alt='assignments' />
+                <Image src={assign}  />
                 <p>ASSIGNMENTS GIVEN</p>
               </div>
+              </Tilt>
             </div>
           </div>
 
+
```
Comment (tech-bash) [inline pages/index.js]: Please remove this empty line

**Comment by tech-bash** [comment]
ok.

**Comment by tech-bash** [comment]
Thanks! @yatigautam for this, hope you make more progress in this project.

**Comment by yatigautam** [comment]
> Thanks! @yatigautam for this, hope you make more progress in this project.

thankyou, ofcourse.

**Comment by gurjeetsinghvirdee** [comment]
add levels on this PR @tech-bash

**Pull request merged**